public/js/constants/atomData.js

```diff
@@ -72,15 +72,27 @@ export const timeline = {
   description: "A series of key events to help readers navigate an ongoing story"
 };
 
-export const allAtomTypes = [cta, recipe, storyQuestions, quiz, media, qa, guide, profile, timeline];
-export const workshopEditableAtomTypes = [cta, storyQuestions, recipe, qa, guide, profile, timeline];
+export const explainer = {
+  type: "explainer",
+  fullName: "Explainer Text",
+  description: "Edit legacy Explainer snippets - (creating new ones is not supported)",
+};
+
+export const allAtomTypes = [cta, recipe, storyQuestions, quiz, media, qa, guide, profile, timeline, explainer];
+export const workshopEditableAtomTypes = [cta, storyQuestions, recipe, qa, guide, profile, timeline, explainer];
 export const snippetAtomTypes = [qa, guide, profile, timeline];
+export const legacyAtomTypes = [explainer];
+
 export function getNonEditableAtomTypes() {
-   return allAtomTypes.filter((atomType) => workshopEditableAtomTypes.indexOf(atomType) === -1);
+   return allAtomTypes.filter((atomType) => {
+     return !workshopEditableAtomTypes.includes(atomType) && !legacyAtomTypes.includes(atomType);
```

Comment:
nice one!

Reply:
If `LegacyAtomType(a) => WorkshopEditableAtomType(a)`, then the `&&` part is not necessary.

Reply:
Just ftr: have decided to keep `legacyAtomTypes` separate from other qualities eg being `workshopEditable` because they don't necessarily overlap. 
Hence the need for the double conditions in this filter statement